const_test.go

```diff
@@ -0,0 +1,100 @@
+package dynamiclistener
```

Comment:
nit: the file name strikes me as weird, there are no tests so the `_test.go` suffix is inappropriate and there are technically no consts. Maybe `test_vars.go` or `test_data.go`?

Reply:
I agree. Maybe move it into a tests directory, and import it in `listener_test.go`?

Reply:
@cmurphy `_test.go`'s intended use is to notify the compiler that a file should only be used when compiling a test. Since these variables are not used outside of tests, I wouldn't want to compile them in all binaries that import dynamiclistener. 

@brandond's suggestion would stop us from compiling in these keys, but I think a test directory without any tests is more confusing than a `vars_test.go` file without any tests.

I am biased because I have seen/used this design pattern for holding large string data that is used only for tests.
I think the alternative is just to paste the variable at the end of the test they are used in, so the file is still readable.

Reply:
I haven't poked at the certs you're embedding here - how long are they valid for? Are we in danger of the automatic renewal code breaking the test when they are within 90 days of expiring?

Reply:
> I haven't poked at the certs you're embedding here - how long are they valid for?

They were valid for 100 years. 

> This library has a ton of cert creation code, Is there any reason we can't just generate the certs as part of the test

I had a long comment here about additional test complexity and not wanting one test for one part of this code to depend on another part, when I decided to just try it to see if I could put something together.

Turns out this worked out pretty well. The test now generates it's own certs instead of using the const. I've eliminated the `const_test` file as a consequence. Thanks @brandond for the suggestion.